pkg/providers/cloudstack/config/template-cp.yaml

```diff
@@ -333,7 +333,7 @@ spec:
 {{- if .kubeletExtraArgs }}
 {{ .kubeletExtraArgs.ToYaml | indent 10 }}
 {{- end }}
-        name: '{{`{{ ds.meta_data.local_hostname }}`}}'
```

Comment:
why are we changing these templates? If we're passing the labels using the WorkerNodeLabels mechanism, why should these change?

Reply:
This is the fix for hostname mismatch between node and CAPI machine.